torchvision/models/vgg.py

```diff
@@ -48,15 +48,15 @@ def _initialize_weights(self):
         for m in self.modules():
             if isinstance(m, nn.Conv2d):
                 n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
-                m.weight.data.normal_(0, math.sqrt(2. / n))
+                nn.init.normal_(m.weight, 0, math.sqrt(2. / n))
```

Comment:
Looking at the implementations in https://github.com/pytorch/pytorch/blob/master/torch/nn/init.py, I think these two lines are equivalent to `kaiming_normal_ (m.weight, mode='fan_out', nonlinearity='relu')`. Could you verify that and then maybe simplify those two lines to only call `nn.init`?

Reply:
If you use `kaiming_normal_ (m.weight, mode='fan_out', nonlinearity='relu')`,  `fan`,`gain` in below is `tensor.size(0)` and `2`. So I think 

```python
std = gain / math.sqrt(fan)
    with torch.no_grad():
return tensor.normal_(0, std)
```

from https://github.com/pytorch/pytorch/blob/master/torch/nn/init.py#L297, is inequivalent to the original.

Reply:
From my understanding, `gain` would be [`math.sqrt(2)`](https://github.com/pytorch/pytorch/blob/master/torch/nn/init.py#L36), and `fan` would be [tensor.size(0) * tensor.size(2) * tensor.size(3)](https://github.com/pytorch/pytorch/blob/master/torch/nn/init.py#L165), which is equivalent to what we have here, right?